guides/source/install_ruby_on_rails.md

```diff
@@ -0,0 +1,133 @@
+**DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
+
+Install Ruby on Rails Guide
+===========================
+
+This guide will walk you through installing the Ruby programming language and the Rails framework on your operating system.
+
+While your OS might come with Ruby pre-installed, it's often outdated and can't be upgraded. Using a version manager like [Mise](https://mise.jdx.dev/getting-started.html) allows you to install the latest Ruby version, use a different Ruby version for each app, and easily upgrade to new versions when they're released.
+
+Alternatively, you can use Dev Containers to run Rails without installing Ruby or Rails directly on your machine. Check out the [Getting Started with Dev Containers](getting_started_with_devcontainer.html) guide to learn more.
+
+--------------------------------------------------------------------------------
+
+## Choose Your Operating System
+
+Follow the section for the operating system you use:
+
+* [macOS](#install-ruby-on-macos)
+* [Ubuntu](#install-ruby-on-ubuntu)
+* [Windows](#install-ruby-on-windows)
+
+TIP: Any commands prefaced with a dollar sign `$` should be run in the terminal.
+
+### Install Ruby on macOS
+
+You'll need macOS Catalina 10.15 or newer to follow these instructions.
+
+For macOS, you'll need Xcode Command Line Tools and Homebrew to install dependencies needed to compile Ruby.
+
+Open Terminal and run the following commands:
+
+```shell
+# Install Xcode Command Line Tools
+$ xcode-select --install
+
+# Install Homebrew and dependencies
+$ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+$ echo 'export PATH="/opt/homebrew/bin:$PATH"' >> ~/.zshrc
+$ source ~/.zshrc
+$ brew install openssl@3 libyaml gmp rust
+
+# Install Mise version manager
```

Comment:
Question about this choice: (And also this guide): 
Ruby comes with this guide itself: https://www.ruby-lang.org/en/documentation/installation/#managers , I am assuming we wanted to give user a simpler/easier way to follow to install Ruby?

Why not prefer mentioning most popular Ruby Version managers from the community: https://railsdeveloper.com/survey/2024/#ruby-rails-version-updates which have better support/community around it for Ruby use?